src/main/java/net/dv8tion/jda/internal/requests/WebSocketClient.java

```diff
@@ -1106,6 +1122,25 @@ else if (requestStage == ConnectionStage.CONNECT)
         return null;
     }
 
+    public void requestMembers(Consumer<DataObject> callback, long guildId, long userId)
+    {
+        synchronized (memberRequests)
+        {
+            Set<Consumer<DataObject>> callbacks = memberRequests.get(guildId);
+            if (callbacks == null)
+                memberRequests.put(guildId, callbacks = new HashSet<>());
+            callbacks.add(callback);
+
+            send(DataObject.empty()
+                .put("op", WebSocketCode.MEMBER_CHUNK_REQUEST)
+                .put("d", DataObject.empty()
+                    .put("guild_id", guildId)
+                    .put("user_ids", Arrays.asList(userId, 0L))
```

Comment:
Why does this array end in 0L?

Reply:
I use it as a marker to check later that the search is meant as a user lookup. The `not_found` will only be present in the payload if an id was not resolved, passing 0 forces it to be present.